rdr_service/services/consent/files.py

```diff
@@ -625,16 +625,15 @@ def _try_to_parse_signature(self, footer_text):
 
     def get_signature_on_file(self):
         signature_offsets = [
-            (8, 10),
-            (73, 75)
+            (8, 15),
+            (73, 80)
         ]
 
         signature_label_strings = [
             "Participant's Name (printed)",
             "'s Name (printed)",
             "Name (printed)",
-            'Nombre del Participant',
-            'Nombre del'
+            'Nombre'
```

Comment:
Use/matching of these strings is always case sensitive, correct?

Reply:
Yeah, but I don't think it would need to be case sensitive. Do you think it would need to be?